src/data/hassio.ts

```diff
@@ -76,12 +84,13 @@ export interface HassioAddonDetails {
   devicetree: boolean;
   docker_api: boolean;
   audio: boolean;
-  audio_input: null | string;
-  audio_output: null | string;
+  audio_input: string;
```

Comment:
I can be `null`

Reply:
![image](https://user-images.githubusercontent.com/5662298/72754749-fea5d400-3bc8-11ea-8037-24997e0c387b.png)